### DISCUSSION TITLE
terrible 4G coverage, will WiFi help me with reliable 80% charge cap?

### DISCUSSION THREAD

**Discussion by nomandera** [flobz/psa_car_controller#948] [Q&A]
As I understand it the psa_car_controller, the Stellantis API and the car all need to be "online" for reliable capping at 80% and other control/lookups.

My issue is that I live in a 4G trouble spot losing connectivity to the car unpredictably and often for hours at a time.

My question.

If I extended WiFi outside so that the car has a permanent strong signal will this be used for control and make my 80% charge cap reliable or is that only for multimedia/map functions?

**Comment by Apollon77** [comment]
Wifi !== 4G ... so the easy answer is ... No

**Reply by nomandera** [reply]
That was my fear. First world problems I suppose but it is frustrating as I have no control at all over 4G (and also I have absolutely amazing 5G coverage).

I will look to maintain it somehow using charger automation.

Edit: Just to be clear after re-reading the reply .... are specifically saying the car wifi (which I have confirmed can connect to my home if drive it much closer) cannot be used for this purpose or was it a general statement that 4G and Wifi are not the same technology (which I understand in excruciating detail). i.e. I have no clue what purpose the cars wifi fulfills currently.

**Reply by nomandera** [reply]
I was wondering if anyone could confirm the above follow up.

The amount of ugly work arounds I am going to do charger side if confirmed is non trivial so I want to make sure before I commit to the effort.
